proto/types/action.proto

```diff
@@ -264,6 +264,10 @@ message Receipt {
   repeated Log logs = 6;
 }
 
+message Receipts {
```

Comment:
not sure if moving message definition between files is a breaking change or not. prefer not to move just in case.